lib/superform/rails.rb

```diff
@@ -74,17 +74,18 @@ def title
         end
       end
 
-      def initialize(model, action: nil, method: nil, **attributes)
+      def initialize(model, action: nil, method: nil, authenticity_token: true, **attributes)
```

Comment:
I've thought about this and I don't think this should be in the constructer. Instead it should be a method that excludes the token if the `method` resolves to `GET`. If the user doesn't want that, they could override the method to:

```
def authenticity_token_field = nil
```